EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||huglivoa.net^$third-party
```

Comment:
How to find this request? I don't see it

Reply:
It's the popup itself.

Reply:
You may not see it, but I see it.
<details>

![image](https://user-images.githubusercontent.com/76908410/110048138-36022080-7d8a-11eb-9eb8-10aa8385237c.png)
</details>